scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -756,7 +772,7 @@ def __init__(self, params):
       raise ValueError('staged_vars for now is only supported with '
                        'variable_update=parameter_server')
 
-    if self.params.variable_update == 'parameter_server':
+    if self.params.variable_update in ('parameter_server', 'horovod'):
```

Comment:
@alsrgv , are we sure we want to use VariableMgrLocalFetchFromPS as variable_mgr in horovod case ? 
Cause (if I'm not missing anything) will use CPU in _build_fetches when placing the initial (before the horovod wraps it) through preprocess_device_grads.
I guess we can use VariableMgrIndependent if there's a case in the above.

Reply:
@shamoya, thanks - good point. Updated.